### PR TITLE
Implement del() on Store interface.

### PR DESCRIPTION
**Pull request by whiten**

### CODE REVIEW

File: src/kv/idbstore.rs [hunk 1]
```diff
@@ -270,15 +270,17 @@ impl WriteTransaction {
 #[async_trait(?Send)]
 impl Read for WriteTransaction {
     async fn has(&self, key: &str) -> Result<bool> {
-        match self.pending.lock().await.contains_key(key) {
-            true => Ok(true),
-            false => self.rt.has(key).await,
+        match self.pending.lock().await.get(key) {
```
Comment (aboodman) [inline src/kv/idbstore.rs:273]: See, you can't say that didn't feel a little good.

File: src/kv/idbstore.rs [hunk 2]
```diff
@@ -294,7 +296,12 @@ impl Write for WriteTransaction {
         self.pending
             .lock()
             .await
-            .insert(key.to_string(), value.to_vec());
+            .insert(key.to_string(), Some(value.to_vec()));
```
Comment (aboodman) [inline src/kv/idbstore.rs:299]: I thought we decided to do `key.into()`?

**Comment by aboodman** [comment]
LGTM

File: src/kv/idbstore.rs (hunk 2 above)
Comment (whiten) [reply src/kv/idbstore.rs:299]: Yeah, we did, thank you. I'll do a quick fixup pass across the codebase now.

Comment (whiten) [reply src/kv/idbstore.rs:299]: https://github.com/rocicorp/repc/commit/90c44f56a807bb6478aa60a992766b736265eb0e

**Pull request merged**